src/Modal.svelte

```diff
@@ -106,6 +106,12 @@
    */
   export let closeOnEsc = true;
 
+  /**
+   * Scrolling behaviour after modal closes
+   * @type {"instant" | "smooth"}
+   */
+  export let scrollBehaviour = undefined;
```

Comment:
Invalid type. I think it'd be better to set it to `auto`.

```suggestion
  export let scrollBehaviour = 'auto';
```